cvat/apps/dataset_manager/tests/utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import tempfile
+import unittest
+from typing import Optional
+
+from datumaro import IDataset
+from datumaro.components.operations import ExactComparator
+from datumaro.util.os_util import rmfile, rmtree
+
+from cvat.apps.dataset_manager.util import current_function_name
+
+
+class FileRemover:
+    def __init__(self, path, is_dir=False):
```

Comment:
Could you add type annotations in the updated code and enable code formatting for the new files?

Reply:
Addede annotations and enabled black and isort for the new file

Reply:
Correction: I can not enable black and isort on CI for the new file alone, because the linters are run on folders, not individual files. Disabled them back

Reply:
It still makes sense to add this file into `dev/format_python_code.sh`

Reply:
Added it there